pghoard/basebackup/delta.py

```diff
@@ -99,6 +99,7 @@ def progress_callback(progress_step: ProgressStep, progress_data: ProgressMetric
                         progress_step.value, progress_data["handled"], elapsed
                     )
                 else:
+                    self.last_flush_time = time.monotonic()
```

Comment:
This would be better at the outer if, no need to have it in each branch. Also can we improve the log messages to better reflect what is happening? Right now it is a bit confusing.